data/upgrades.js

```diff
@@ -2545,7 +2585,17 @@
     ],
     "id": 221,
     "xws": "tieshuttle",
-    "image": "upgrades/Title/tie-shuttle.png"
+    "image": "upgrades/Title/tie-shuttle.png",
+    "grants": [
+      {
+        "type": "slot",
+        "name": "Crew"
+      },
+      {
+        "type": "slot",
+        "name": "Crew"
+      }
+    ]
```

Comment:
Mhm perhaps we should also have a way to define the slots that get removed. The TIE Shuttle card should remove all Torpedo, Missile and Bomb slots.